en/file_formats/mavlink.md

```diff
@@ -0,0 +1,41 @@
+# Log File Formats
+
+*QGroundControl* supports two log file formats:
+
+* Plain MAVLink packet logs to "replay" a previous mission.
+* Comma-separated values (CSV) for further analysis and visualization (e.g. into Matlab, R, Mathematica, etc)
```

Comment:
As far as I know CSV is not supported

Reply:
I'll remove that then and merge.  Thanks!